src/main.rs

```diff
@@ -315,10 +336,10 @@ fn switch_off_leds() {
 
 fn check_user_presence(cid: ChannelID) -> Result<(), Ctap2StatusCode> {
     // The timeout is N times the keepalive delay.
-    const TIMEOUT_ITERATIONS: usize = ctap::TOUCH_TIMEOUT_MS as usize / KEEPALIVE_DELAY_MS as usize;
+    const TIMEOUT_ITERATIONS: u32 = ctap::TOUCH_TIMEOUT_MS / KEEPALIVE_DELAY_MS;
```

Comment:
Is this conditional on the architecture? So it should actually be u32 on a 32 bit machine etc.? Then it's closer in intention to the `usize`. Though the compiler doesn't accept that as easily.

Reply:
I changed it to ClockInt for consistency.